src/Rtc/Join.tsx

```diff
@@ -25,13 +30,25 @@ const Join: React.FC<{
       }
     }
     const videoState = uidState.max[0].video;
+
     async function join() {
+      /* Live Streaming */
+      if (rtcProps.mode === mode.Live) {
+        // await engine.setChannelProfile(ChannelProfile.LiveBroadcasting);
```

Comment:
Why is `setChannelProfile` commented out?

Reply:
Implemented, Is available in the latest code